template/bsconfig.json

```diff
@@ -1,30 +1,21 @@
 {
   "name": "my-rescript-app",
   "reason": {
-    "react-jsx": 3
+      "react-jsx": 3
   },
   "sources": {
-    "dir" : "src",
-    "subdirs" : true
+      "dir": "src",
+      "subdirs": true
   },
-  "bsc-flags": [
-    "-bs-super-errors",
-    "-bs-no-version-header"
-  ],
+  "bsc-flags": ["-bs-super-errors", "-bs-no-version-header"],
   "package-specs": [
-    {
-      "module": "es6",
-      "in-source": true
-    }
+      {
+          "module": "commonjs",
```

Comment:
Why do we need to prefer commonjs output over esm?

Reply:
In my case, while I was trying to setup the template and verify if it was working, I couldn't get it to compile unless module was set to `commonjs`. To be honest I looked for the reason, but I have no idea why it wasn't working
But maybe it was specific to my setup and I can put back `es6`